### PR TITLE
Add statistics and maproulette task generation

### LINKED ISSUE #113: run stats generation
stats has been moved out from tile cutting so we'll need to run it one off like connectivity. this is nice because we dont have to do the kludge of finding and moving it with a date.

### PR DESCRIPTION
**Pull request by Kopkins**
fixes #113

### CODE REVIEW

File: templates/default/cut_tiles.sh.erb [hunk 1]
```diff
@@ -61,8 +61,10 @@ cur_extras_dir=<%= node[:valhalla][:base_dir] %>/extras_${stamp}
 mkdir -p ${cur_extras_dir}
 pushd ${cur_extras_dir}
 valhalla_build_connectivity -c <%= node[:valhalla][:config] %>
+valhalla_build_statistics -c <%= node[:valhalla][:config] %>
 for f in connectivity*; do  mv_stamp $f ${stamp}; done
-mv_stamp $(jq -r '.mjolnir.statistics' <%= node[:valhalla][:config] %>) ${stamp}
+mv_stamp statistics.sqlite ${stamp}
+mv_stamp tasks.json ${stamp}
```
Comment (kevinkreiser) [inline templates/default/cut_tiles.sh.erb]: name of `tasks.json` should change to something a bit more descriptive. we can merge this as soon as we do that and get i merged in mjölnir

Comment (Kopkins) [inline templates/default/cut_tiles.sh.erb]: This just has to match the maproulette_tasks field in `conf/valhalla.json`

Comment (kevinkreiser) [inline templates/default/cut_tiles.sh.erb]: based on your valhalla/mjolnir#501 shouldnt this be `mv_stamp maproulette_tasks.geojson ${stamp}`

**Comment by kevinkreiser** [comment]
:rocket:

**Pull request merged**